beluga_amcl/src/amcl_node.cpp

```diff
@@ -931,7 +931,10 @@ void AmclNode::laser_callback(ExecutionPolicy&& exec_policy, sensor_msgs::msg::L
           get_parameter("laser_max_range").as_double(),
       }
               .points_in_cartesian_coordinates() |
-          ranges::views::transform([](const auto& value) { return std::make_pair(value.x(), value.y()); }) |
+          ranges::views::transform([&](const auto& p) {
+            const auto result = base_to_laser_transform * Sophus::Vector3d{p.x(), p.y(), 0};
```

Comment:
nit: good opportunity to rename to `laser_pose_in_base`.

Reply:
Definitely, done in https://github.com/Ekumen-OS/beluga/pull/276/commits/04a91463ec39ceed3ee673d828cb44c5f66a3f50.